python/lsst/afw/geom/ellipses/PixelRegion.i

```diff
@@ -59,12 +59,15 @@
 
 %extend PyPixelRegionIterator {
 %pythoncode %{
-def next(self):
+def __next__(self):
     if self.atEnd():
         raise StopIteration()
     current = self.get()
     self.increment()
     return current
+
+next = __next__
```

Comment:
What is this for? Why not use `from builtins import object`? Per http://python-future.org/compatible_idioms.html#custom-iterators